csharp/SparkCLR.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.SparkCLR</id>
+    <version>1.0.0</version>
+    <authors>Kaarthik Sivashanmugam, Renyi Xiong</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>https://github.com/Microsoft/SparkCLR/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/Microsoft/SparkCLR</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>C# language binding and extensions to Apache Spark</description>
+    <releaseNotes>See release notes on GitHub https://github.com/Microsoft/SparkCLR/releases</releaseNotes>
+    <copyright>Copyright 2015</copyright>
```

Comment:
I have pushed an update to this PR branch to amend the <copyright> element to match the copyright header in the code, as @skaarthik suggested.